src/server/database/getRethinkConfig.js

```diff
@@ -0,0 +1,34 @@
+import {readCert} from './readCert';
+import flag from 'node-env-flag';
+
+export const getRethinkConfig = () => {
+
+  let config = {
+    host: process.env.DATABASE_HOST || 'http://rethinkdb.dev.azk.io' || 'localhost',
```

Comment:
Some dead code - impossible to hit 'localhost'. A string will always by thruthy, therefore 'http://rethinkdb.dev.azk.io' will always be the result if DATABASE_HOST is not set.